modules/visualization/dcv-k8s-deployment/README.md

```diff
@@ -0,0 +1,66 @@
+# DCV-EKS Module
+
+## Description
+
+This module helps running the `visualization/dcv-image` module ECR image. It runs as a k8s deployment on GPU nodes using the k8s concepts of taints/toleration and node labels.
+
+Once the deployment is completed, the app is exposed using a k8s Service of type NLB and you can accept it using the below address:
+
+```text
+https://<<NLB-DNS>>:8443
+```
+
+Today, the DCV sessions are not created automatically. Pls reach out to `Aubrey` from AWS Proserv and he can help setup the DCV sessions
```

Comment:
Remove comment

Reply:
Looks like the Dockerfile -> run_script.sh -> startup_script.sh which should start the session

Reply:
Yeah. That isn't happening and would appreciate your help on that part.